addons/l10n_in/views/report_invoice.xml

```diff
@@ -56,6 +57,8 @@
                 <span t-elif="o.move_type == 'in_invoice'">Vendor Bill</span>
                 <span t-if="o.name != '/'" t-field="o.name"/>
             </h2>
+        </t>
+        <t t-else="">$0</t>
```

Comment:
@jco-odoo Here is the issue you are just replacing `<h2>` tag with `<t>`? and this just doesn't seems like a hardcore fix for l10n_in we need to fix this issue from the start with a new PR. Because this fix is buggy, I suggest we can do something like done in l10n_au but need to make sure we do it for every l10n_in (edi, ewaybill)

Still I don't get why $0?
```xml
        <t t-else="">$0</t>
```